test/TestFormatter.py

```diff
@@ -29,7 +30,7 @@ class TestFormatter(unittest.TestCase):
     def setUp(self):
         self.rule = AnsibleLintRule()
         self.rule.id = "TCF0001"
-        self.formatter = Formatter()
+        self.formatter = Formatter(os.getcwd(), True)
```

Comment:
How about we get rid of `os`?
```suggestion
        self.formatter = Formatter(pathlib.Path.cwd(), True)
```